dashboard/app/reporting_email.go

```diff
@@ -239,17 +246,23 @@ func emailReport(c context.Context, rep *dashapi.BugReport) error {
 	if err := json.Unmarshal(rep.Config, cfg); err != nil {
 		return fmt.Errorf("failed to unmarshal email config: %v", err)
 	}
-	to := email.MergeEmailLists([]string{cfg.Email}, rep.CC)
+	to := []string{}
+	cc := email.MergeEmailLists([]string{cfg.Email}, rep.CC)
+	if len(rep.Recipients) > 0 {
```

Comment:
This looks similar to `emailSendBugNotif()`.  Can any code be shared?

Reply:
There are only a few similar lines and they handle different structs but they can be shared. I can create a function that receives all different sources of emails and conditions and returns to and cc. Do you think that's better?

Reply:
Well, maybe.  My instinct is that this logic should be shared.  Although looking at it more closely, it's not trivial to share.  We could do:

        func buildEmailLists(cfg *EmailConfig, public bool, recipients RecipientInfo[],
                             maintainers, legacyCC string[]) (to, cc string[]) {
        }

Not sure it's worthwhile though -- that's a lot of parameters.  I'll leave it up to you...